common/Mesh-Network-Joining-Process-Comparison.md

```diff
@@ -0,0 +1,540 @@
+
+### 2.3.2. Establish a New Network
+1. The procedure to establish a new network is initiated through use of the *NLME-NETWORK-FORMATION.request* primitive, the primitive allows the next higher layer to request that the device start a new ZigBee network with itself as the coordinator. Only the device which is capable of becoming the Zigbee coordinator and not currently joined to a network shall attempt to establish a new network.    
+
+2. When this procedure is initiated, the Network Layer Management Entity (NLME) shall first request that the MAC sub-layer(s) perform an energy detection scan over either a specified set of channels or, by default, the complete set of available channels, as dictated by the PHY layer(s), to search for possible interferers. In EmberZNet stack based project, you can configure the channel mask in the **Network Creator** plugin. And of course, the energy detection scan is not necessary if there is only one channel specified.   
+On receipt of the results from a successful energy detection scan, the NLME shall order the channels on each interface according to increasing energy measurement and discard those channels whose energy levels are beyond an acceptable level.   
+
+3. The NLME shall then perform an active scan, by issuing the *MLME-SCAN.request* primitive on each MAC Interface with the ScanType parameter set to active scan and ChannelList set to the list of acceptable channels and ChannelPage set to the relevant value for that interface, to search for other ZigBee devices. To determine the best channel on which to establish a new network, the NLME shall review the list of returned PAN descriptors and find the first channel for each MAC Interface with the lowest number of existing networks, favoring a channel with no detected networks.   
+
+4. If a suitable channel is found, the NLME shall select a PAN identifier for the new network. To do this the device shall choose a random PAN identifier less than 0xffff that is not already in use on the selected channel.   
+Once a PAN identifier is selected, the NLME shall select a 16-bit network address equal to 0x0000 as its network address. And then the NLME shall check the value of the extended PAN identifier for the PAN. If this value is 0x0000000000000000 means the extended PAN identifier is unknown, this attribute will be initialized as well.   
+
+5. Then, the NLME shall initiate a new PAN by issuing the *MLME-START.request* primitive to each MAC sub-layer. And each MAC interface will response the status of the PAN startup via the *MLME-START.confirm* primitive.   
+6. On receipt of the status of the PAN startup, the NLME shall inform the next higher layer of the status of its request to initialize the ZigBee coordinator.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-establish-network.png">  
```

Comment:
1. [Mark]Can we put this image on top of the description. It was a hard time to understand the long description without seeing the image.
2. [Mark] I haven't had a chance to look at rest part of section 2. It looks too detailed to me which might need Jim to put comments.